eng/pipelines/runtime.yml

```diff
@@ -118,6 +118,44 @@ jobs:
       - eng/pipelines/mono/*
       - eng/pipelines/libraries/*
 
+#
```

Comment:
are both checked and retail flavors required for corelib, or are they essentially the same in both flavors?

Reply:
The CoreLib flavor has to match the CoreCLR flavor.

Reply:
Note that you can use `platform-matrix-multijob` to make this simpler. You can do something like:

```yml
- template: /eng/pipelines/common/platform-matrix-multijob.yml
  parameters:
    platforms:
    - Windows_NT_x64
    - Linux_x64
   jobTemplates:
   - jobTemplate: /eng/pipelines/coreclr/templates/build-corelib-job.yml
     buildConfig: checked
     jobParameters:
       testGroup: inerloop
   - ...
    More job templates
   - ...
  
```

That way you share the platform matrix across multiple jobs to keep in sync.

Reply:
thanks for the tip, will update once we get the basic parallelization working.

Reply:
I am going to defer this optimization to a subsequent change.